src/Domain/MigrationStep/s050_DestinationPimInstallation/DestinationPimVersionChecker.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\PimMigration\Domain\MigrationStep\s050_DestinationPimInstallation;
+
+use Akeneo\PimMigration\Domain\Command\ChainedConsole;
+use Akeneo\PimMigration\Domain\Command\SymfonyCommand;
+use Akeneo\PimMigration\Domain\Pim\DestinationPim;
+
+/**
+ * Checks that the destination has all the required end-points
+ *
+ * @author    Laurent Petard <laurent.petard@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ */
+class DestinationPimVersionChecker
+{
+    const EXACT_MAJOR_VERSION = 2;
+    const EXACT_MINOR_VERSION = 0;
+    const MINIMUM_PATCH_VERSION = 3;
```

Comment:
Ok, now I understand the READ.me but you still should enhanced the read.me.

Reply:
I'm really bad to write some sort of sentence. Do you have a idea how to say that clearly ?